### ISSUE TITLE
Exception while starting my program

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ankit4u3/jtracert#3]
```
What steps will reproduce the problem?
1. Just start your program. I used the following command:
java -javaagent:jTracert.jar com.eibus.soap.Processor -dn "cn=Notification 
Processor,cn=Notification Service,cn=soap 
nodes,o=system,cn=cordys,cn=defaultinst,o=vanenburg.com"


What is the expected output? What do you see instead?
PROCSTOP:-
11:Y249Tm90aWZpY2F0aW9uIFByb2Nlc3Nvcixjbj1Ob3RpZmljYXRpb24gU2VydmljZSxjbj1z
b2FwIG5vZGVzLG89c3lzdGVtLGNuPWNvcmR5cyxjbj1kZWZhdWx0aW5zdCxvPXZhbmVuYnVyZy5
jb20=:com/google/code/jtracert/traceBuil
der/MethodCallTraceBuilderFactorycom.sun.crypto.provider.SunJCE.<clinit>
(DashoA13*..)sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native 
Method)sun.reflect.NativeConstructorAccessorImpl.newI
nstance(NativeConstructorAccessorImpl.java:39)
sun.reflect.DelegatingConstructorAccessorImpl.newInstance
(DelegatingConstructorAccessorImpl.java:27)
java.lang.reflect.Constructor.newInstance(Constructor.
java:513)java.lang.Class.newInstance0(Class.java:355)
java.lang.Class.newInstance(Class.java:308)
sun.security.jca.ProviderConfig$3.run(ProviderConfig.java:240)
java.security.AccessController.doPrivilege
d(Native Method)sun.security.jca.ProviderConfig.doLoadProvider
(ProviderConfig.java:225)sun.security.jca.ProviderConfig.getProvider
(ProviderConfig.java:205)sun.security.jca.ProviderList.loadAll(Provide
rList.java:264)sun.security.jca.ProviderList.removeInvalid
(ProviderList.java:281)sun.security.jca.Providers.getFullProviderList
(Providers.java:158)java.security.Security.insertProviderAt(Security.java
:325)java.security.Security.addProvider(Security.java:372)
com.eibus.soap.Processor.<init>(Processor.java:313)
com.eibus.soap.Processor.startProcessor(Processor.java:1021)
com.eibus.soap.ProcessStreamsHa
ndler.startProcessor(ProcessStreamsHandler.java:653)
com.eibus.soap.ProcessStreamsHandler.startProcessor
(ProcessStreamsHandler.java:592)com.eibus.soap.Processor.main
(Processor.java:1353)

What version of the product are you using? On what operating system?
jTracert-0.0.3

Please provide any additional information below.


```

Original issue reported on code.google.com by `spatnai...@gmail.com` on 10 Oct 2008 at 6:25

**Comment by GoogleCodeExporter** [comment]
```
There must be a problem when instantiating SunJCE. This class causes the same 
failure:

public class TestJtracert
{

  public static void main(String[] args)
  {
    com.sun.crypto.provider.SunJCE des = new com.sun.crypto.provider.SunJCE();
  }

}

Exception in thread "main" java.lang.NoClassDefFoundError:
com/google/code/jtracert/traceBuilder/MethodCallTraceBuilderFactory
        at com.sun.crypto.provider.SunJCE.<clinit>(DashoA12275)
        at TestJtracert.main(TestJtracert.java:6)

```

Original comment by `almohadi...@gmail.com` on 22 Dec 2008 at 10:56

**Comment by GoogleCodeExporter** [comment]
```
I can reproduce this defect - looks like jTracert incorrectly handles classes 
loaded
with custom classloaders.
Investigating...
```

Original comment by `Dmitry.Bedrin` on 6 Jan 2009 at 9:56
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in SVN.

Actually I've committed a temporary fix for now - com.sun.crypto.provider.SunJCE
class isn't instrumented or analyzed any more.
```

Original comment by `Dmitry.Bedrin` on 6 Jan 2009 at 10:13
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
When is the next release?


```

Original comment by `spatnai...@gmail.com` on 7 Jan 2009 at 8:48

**Comment by GoogleCodeExporter** [comment]
```
I can't tell right now actually. Perhaps in a couple of weeks - I still have 
things
to implement for this version.

You can build jTracert from SVN yourself - I've described it here:
http://code.google.com/p/jtracert/wiki/BuildingJTracertFromSVN
```

Original comment by `Dmitry.Bedrin` on 7 Jan 2009 at 9:07

**Comment by GoogleCodeExporter** [comment]
```
Reproduced in SimpleApp2. Fixed in revision r96
```

Original comment by `Dmitry.Bedrin` on 27 Jan 2009 at 1:45

**Comment by GoogleCodeExporter** [comment]
```
Fixed in jTracert 0.0.4
```

Original comment by `Dmitry.Bedrin` on 28 Jan 2009 at 8:36

**Comment by GoogleCodeExporter** [comment]
Original comment by `Dmitry.Bedrin` on 6 Feb 2009 at 12:39
- Changed state: **Verified**
